parallel_benchmark_test.go

```diff
@@ -0,0 +1,57 @@
+package roaring
+
+import (
+	"math/rand"
+	"testing"
+)
+
+func BenchmarkIntersectionLargeParallel(b *testing.B) {
```

Comment:
I'm impressed that there are any benefits whatsoever to parallelize the intersection. That's great. But the reason I am impressed is that intersections are not what kills you, typically... the performance troubles are almost always with unions, those are tremendously expensive at times.  So, I wonder, why did you write a benchmark for intersections, but not for unions (ParOr)?

Reply:
In the private benchmark I referred to in #107 the improvement is clearly visible:

Par:
```
and:24.346914ms, or:106.765985ms, andNot:7.937437ms
```

Fast:
```
and:49.893867ms, or:448.947547ms, andNot:7.49725ms
```

running on a quad-core 2013' MacBook Pro, Go 1.9.1.

I'm thinking of writing code that would generate similar benchmark data, however I'd prefer to open a separate ticket for that and have it done later if that's OK with you.

Reply:
Ok.